app/javascript/packages/document-capture/components/file-input.tsx

```diff
@@ -388,7 +400,7 @@ function FileInput(props: FileInputProps, ref: ForwardedRef<any>) {
             onClick={onClick}
             onDrop={onDrop}
             accept={accept ? accept.join() : undefined}
-            aria-describedby={hint ? hintId : undefined}
+            aria-describedby={shouldShowHint() ? hintId : undefined}
```

Comment:
Do we need this change? I'm having a hard time following how `shouldShowHint` controls both the `aria-describedby` + `hintId` as well as the `aria-labelledby` + `innerHintId` above, where it seems like at least in this instance, the element with `hintId` is always present as long as `hint` is truthy, which is what the previous logic was already doing.

Reply:
it is not needed, dropped it